app/components/TrackCard/index.js

```diff
@@ -53,10 +94,13 @@ export function TrackCard({
         </If>
       </Flex>
       <If
-        condition={!isEmpty(artistName)}
+        condition={!isEmpty(previewUrl)}
         otherwise={<T data-testid="preview-audio-unavailable" id="preview_audio_unavailable" />}
       >
-        <audio src={previewUrl} controls />
+        <Button data-testid="preview-audio-button" onClick={handleOnPlay} type="primary">
+          {isPlaying ? 'Pause preview' : 'Play Preview'}
+        </Button>
+        <audio data-testid="preview-audio" ref={audioRef} src={previewUrl} />
```

Comment:
Audio controls should show up

Reply:
The button present above is what controls the controls for the audio element, therefore the controls in the audio element itself are disabled.